packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -92,15 +92,16 @@ library AncillaryData {
         bytes memory key,
         uint256 value
     ) internal pure returns (bytes memory) {
-        bytes memory prefix = _appendKey(currentAncillaryData, key);
+        bytes memory prefix = constructPrefix(currentAncillaryData, key);
         return abi.encodePacked(currentAncillaryData, prefix, toUtf8BytesUint(value));
     }
 
     /**
-     * @notice Helper method that returns the LHS of a "key:value" pair plus the colon ":" and a leading comma "," if
```

Comment:
- In the inline docs for `AncillaryData`, ["LHS"](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/common/implementation/AncillaryData.sol#L100) should be written out as "left hand side".